libs/blocks/notification/notification.js

```diff
@@ -99,7 +102,27 @@ function decorateFlexible(el) {
   el.appendChild(inner);
 }
 
-function decorateLayout(el) {
+async function loadIconography() {
+  await new Promise((resolve) => { loadStyle(`${base}/styles/iconography.css`, resolve); });
+}
+
+async function decorateLockup(lockupArea, el) {
+  await loadIconography();
+  const icon = lockupArea.querySelector('picture');
+  const content = icon.nextElementSibling || icon.nextSibling;
+  const label = createTag('span', { class: 'lockup-label' }, content.nodeValue || content);
+  if (content.nodeType === 3) {
+    lockupArea.replaceChild(label, content);
+  } else {
+    lockupArea.appendChild(label);
+  }
+  lockupArea.classList.add('lockup-area');
```

Comment:
Do you have an example of this being used? `lockup-area`. I'm wondering how the iconography.css is scoped to this block as I don't see it being used in the examples? maybe i'm blind. :) 
| Added support for product lockups in Notifications

Reply:
D'oh, sorry, you're right -- I added it to the description but here's the page Vicky was looking at to test all the new things: https://ebartholomew-general-notification-fixes--milo--adobecom.hlx.page/drafts/vmorel/red-team/notification/mwpw-155529